ui/Screen/Org/Projects.svelte

```diff
@@ -33,6 +33,22 @@
       urn: project.urn,
     });
   };
+
+  let pendingResolved: project.Project[];
+  let confirmedResolved: project.Project[];
+  let pendingUnresolved: project.Anchor[];
+  let confirmedUnresolved: project.Anchor[];
+
+  $: {
+    pendingResolved = anchoredProjects.filter(
+      p => p.anchor && p.anchor.type === "pending"
+    );
+    confirmedResolved = anchoredProjects.filter(
+      p => p.anchor && p.anchor.type === "confirmed"
+    );
+    pendingUnresolved = unresolvedAnchors.filter(a => a.type === "pending");
+    confirmedUnresolved = unresolvedAnchors.filter(a => a.type === "confirmed");
+  }
```

Comment:
This should all go into the `org` module: daef4a96da3729437c930eaaf7aa806f65b8dde6.